app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -397,7 +540,15 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_configChange_lastWeekSectionTitleIsDisplayed() {
+  fun testRecentlyPlayedTestActivity_configChange_lastWeekSectionTitleIsDisplayed() { // ktlint-disable max-line-length
```

Comment:
/ ktlint-disable max-line-length not required i guess

Reply:
Removed.